### PR TITLE
Fix: UI improvements in navbar, status, compare page.

### PR DESCRIPTION
**Pull request by whatDeepak**
This PR enhances the responsiveness and user experience of the application.

1. Navbar Responsiveness: I've made updates to the navbar component to ensure it adapts smoothly to various screen sizes. Now, users can easily navigate the application regardless of the device they're using.
2. Responsive Status Page: The status page has overall responsiveness improvements.
3. Enhanced Compare Page: The input bar on the compare page has been redesigned to improve its aesthetics and user-friendliness. Now, it aligns better with the overall design of the application and provides a smoother input experience as it is responsive. ( The earlier design had some movements which was not that good UX experience. If the newer one doesn't work for you, I'll be more than happy to redesign. )

**Issue Number:**

-  Related to #525 
- Fixes https://github.com/vitessio/arewefastyet/issues/551

**Screenshots/videos:**

Before:

Frame 3

After: 

I believe these changes will significantly enhance the overall usability and accessibility of the application, contributing to a more positive user experience.

Additionally, I'd like to request access to secret key necessary to further contribute to the project. Having access to these resources will enable me to continue making meaningful contributions to the project's development.

 I'm looking forward to your feedback and suggestions for further improvements. :)

### CODE REVIEW

**Comment by whatDeepak** [comment]
hey @frouioui ! Can you share your views on this pls and guide me further.

File: website/src/pages/ComparePage/components/Hero.tsx
Comment (Ari1009) [inline website/src/pages/ComparePage/components/Hero.tsx:1]: Hey, I made a similar pull request [HERE](https://github.com/vitessio/arewefastyet/pull/546) Can you wait for its approval before making any commits on this section?

File: website/yarn.lock
Comment (Ari1009) [inline website/yarn.lock:1]: The yarn.lock changes affect everyone locally, so there is no need to commit and push this.

**Review by Ari1009 — CHANGES_REQUESTED** [review]
Hey @whatDeepak, can you check out the file comments? Thanks!

File: website/src/pages/ComparePage/components/Hero.tsx
Comment (whatDeepak) [reply website/src/pages/ComparePage/components/Hero.tsx:1]: sure, i'll not be making any further commits here and wait till yours get approved.

File: website/yarn.lock
Comment (whatDeepak) [reply website/yarn.lock:1]: thenks, i actually was of the impression that it needs to be pushed for consistency. (Few articles states that) I'll keep that in mind. thanks again for the heads up :)

Comment (Ari1009) [reply website/yarn.lock:1]: If you haven't made any dependency changes, there's no need to commit and push the yarn.lock file.

File: website/src/pages/ComparePage/components/Hero.tsx
Comment (frouioui) [inline website/src/pages/ComparePage/components/Hero.tsx:36]: In my opinion we should keep the same style for the input as what we have on other pages, here is a diff between search and compare:

image
image

**Review by frouioui — COMMENTED** [review]
Left a review that should be addressed, otherwise this is a great contribution, thanks a bunch 🙇🏻

File: website/src/pages/ComparePage/components/Hero.tsx
Comment (whatDeepak) [reply website/src/pages/ComparePage/components/Hero.tsx:36]: got it, I'll make the input style consistent across pages 
what i suggest: instead of keeping separate input boxes, i'll follow the old style only and make that responsive by decreasing the size of that in small devices

Comment (whatDeepak) [reply website/src/pages/ComparePage/components/Hero.tsx]: hey @frouioui , kindly check if this is fine by you and addresses the review comments correctly?

this fix brings responsiveness to compare page while keeping the input style consistent with older design.

Before : 

After :

**Comment by frouioui** [comment]
Hello again @whatDeepak, truly thank you for taking the time to work on this. @Jad31 recently revamped these pages/components, I will then close this PR for now.

**Pull request closed without merging**
